tests/playbooks/tests_bridge_cloned_mac.yml

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Test configuring bridges
+  hosts: all
+  tasks:
+    - name: "define vars"
+      set_fact:
+        interface: "LSR-TST-br31"
+        cloned_mac: "12:23:34:45:56:70"
+
+- name: Add test bridge
+  hosts: all
+  vars:
+    network_connections:
+      - name: "{{ interface }}"
+        interface_name: "{{ interface }}"
+        state: up
+        type: bridge
+        ip:
+          dhcp4: no
+          auto6: yes
+        cloned_mac: "{{ cloned_mac }}"
+  roles:
+    - linux-system-roles.network
+  tasks:
+    - name: Verify ifcfg MACADDR entry
+      command: >-
+        grep ^MACADDR=
+        /etc/sysconfig/network-scripts/ifcfg-{{ bridge_interface }}
```

Comment:
The CI failed. The error was: 'bridge_interface' is undefined.

Should this be ? 
```
        grep ^MACADDR=
        /etc/sysconfig/network-scripts/ifcfg-{{ interface }}
```

Reply:
don't understand how i commit that but yes definitly an error